src/main/java/com/utilaider/logistics/controller/LogisticsController.java

```diff
@@ -142,21 +188,15 @@ public ModelAndView completeProfile(@ModelAttribute Owner owner, BindingResult r
                 fetchedOwner.setNoOfVehicles(owner.getNoOfVehicles());
                 fetchedOwner.setBusinessType(owner.getBusinessType());
                 fetchedOwner.setUserReferanceCode(owner.getUserReferanceCode());
-                owner.getAddress().setPincode(Integer.parseInt(request.getParameter("pincode111")));
+                owner.getAddress().setUser(fetchedOwner);
```

Comment:
This is bi-directional relationship
I will use shared primary key.
In Address class User class use as primary key